hub-database.sh

```diff
@@ -87,12 +102,87 @@ else
   exit $exitCode
 fi
 
+## Client Certificate management - Used to inter-database foreign data access
+$hubDatabaseDir/bin/certmanager.sh db-client-cert --ca $targetCAHost:$targetCAPort --rootcert $PGDATA/root.crt --key $PGDATA/hub-db-user.key --cert $PGDATA/hub-db-user.crt --outputDirectory /tmp --commonName blackduck_user --autocopy false
+exitCode=$?
+if [ $exitCode -eq 0 ];
+then
+
+  # Convert naming to agnostic standard
+  if [ -f /tmp/blackduck_user.crt ]
+  then
+  	echo "Renaming client certificate"
+  	
+  	# Convert client key to usable format
+    mv /tmp/blackduck_user.crt /tmp/hub-db-user.crt
+    rm -f /tmp/blackduck_user.crt
```

Comment:
Why `rm` if you do `mv`?

Reply:
To keep the file state consistent - I don't want to leave errant files around. Unless... wait, mv is effectively rename, isn't it (facepalm)

Reply:
Extra "rm"'s removed